{{cookiecutter.project_slug}}/.github/workflows/expo-build.yml

```diff
@@ -0,0 +1,66 @@
+# Manually triggered workflow for creating a build
+name: Expo Build with given profile
+
+on:
+  workflow_dispatch:
+    inputs:
+      profile:
+        description: "Build profile"
+        required: true
+        default: "development"
+        type: choice
+        # Should match the ones in `eas.json`
+        options:
```

Comment:
this goes nicely with my open PR as well that creates dynamic review app based builds. 
I think dynamically adding them to the eas.json like this allows us to create unique builds. In this case as you have it, each engineer's build will overwrite the others (although at this moment many of us don't really need builds that often). 
I also wonder if this can be avoided by removing the channel arg

Reply:
I mean initially I wrote this because I happened not to have enough permissions in a project to use the eas cli myself so I just used the action to get a build  I needed.
This as you said is not something that we would use often, it is just a convenience, mostly to get development builds that we usually need when we deviate from expo go.